app/assets/javascripts/redux/modules/models/promotionalTweets.js

```diff
@@ -17,6 +25,25 @@ export const initialState = {
   loading: false,
 };
 
+function handlePromoTweetSuccessResponse(state, action) {
+  const normalizedPromoTweet = normalizeListenSignalsResponse(action.payload);
```

Comment:
this is f'ed up. 

action.payload data is getting populated under object['result'] rather than object['entities']...

@ricardo-quinones